src/renderer/components/drawer/drawer.tsx

```diff
@@ -125,9 +137,21 @@ export class Drawer extends React.Component<DrawerProps> {
     if (open) onClose();
   };
 
+  copyK8sObjName = () => {
+    const { title } = this.props;
+    const k8sObjName = title.toString().split(': ')[1];
+    clipboard.writeText(k8sObjName);
+    this.setState({copyIcon: "done", isCopied: true});
+    setTimeout(() => {
+      this.setState({copyIcon: "content_copy", isCopied: false});
+    }, 400);
+  };
+
   render() {
     const { open, position, title, animation, children, toolbar, size, usePortal } = this.props;
     let { className, contentClass } = this.props;
+    const { isCopied, copyIcon } = this.state;
+    const copyTooltip = isCopied? "Copied!" : "Copy";
```

Comment:
We can create also `copyIcon` constant here to keep component state clean (only with 1 value `isCopied`).